QuizApp/QuizApp/CoordinatorProtocol.swift

```diff
@@ -4,4 +4,6 @@ protocol CoordinatorProtocol {
 
     func showUserViewController()
 
+    func logOut()
```

Comment:
This is too vague for Coordinator functions. They should all start with `show` and indicate what page will be shown.